store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -442,6 +443,9 @@ public boolean flush(final int flushLeastPages) {
         if (mappedFile != null) {
             long tmpTimeStamp = mappedFile.getStoreTimestamp();
             int offset = mappedFile.flush(flushLeastPages);
+            if (mappedFile.getflushError()) {
```

Comment:
If we got error during flush, is it better not forwarding `flushedWhere` below.?